src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -131,6 +136,7 @@ const ConsequencesTableComponent = compose(
       style={{ width: '100%', minWidth: 450 }}
       data={dataRows}
       prefix={paginationPrefix}
+      entityName={'entries'}
```

Comment:
That would also work

I just noticed though that we do refer to these as entires in the pagination size selection, which is used throughout the portal 
![image](https://user-images.githubusercontent.com/743976/27976562-10f60054-6335-11e7-907e-9e34eb6f25c7.png)